src/ui/profile_preferences.ui

```diff
@@ -1032,7 +1049,7 @@ you can use it but there could be issues with aligning columns of text</string>
               </widget>
              </item>
              <item>
-              <widget class="QLabel" name="label_23">
+              <widget class="QLabel" name="label_wrapCharacters">
```

Comment:
:+1: for giving a meaningful name to the label for the control - too many of them have been left at the default of `label_` and a number...! :grinning: